### PR TITLE
Swift Style Guide

### PR DESCRIPTION
**Pull request by jTanG0506**
### Description of the Change
Adds a start for a Swift styling guide, based on the existing file in the iOS folder. Also fixes the broken 'Contributing Guidelines' link on the main README.

### CODE REVIEW

File: iOS/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<p align="center">
```
Comment (wordshaker) [inline iOS/README.md]: Is it worth having Swift in it's on markdown file and linking to it from the main readme (could even have an internal readme as a contents if you wanted as well).

I don't know much about the word of iOS so don't know how many guidelines would be included in this folder. @simongilmurray may have more input. I've asked him to PR the contents of this work as well.

File: iOS/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    case ARedEnum = 1
+    case ABlueEnum = 2
+}
+```
+
+### Type Inference
+Let the compiler infer the type for constants. Only specify the specific type when required, such as `CGFloat`.
+#### Preferred:
+```
+let name: String = "UNiDAYS"
```
Comment (jtthrillson) [inline iOS/README.md]: This line doesn't seem to be following the rules immediately above it.

Comment (jTanG0506) [reply iOS/README.md]: Oops. Made requested changes.

**Comment by jtthrillson** [comment]
The changes looks good.

File: iOS/README.md (hunk 1 above)
Comment (jTanG0506) [reply iOS/README.md]: @wordshaker Made requested changes and moved it to Swift folder incase any future guides want to be added to iOS but is not necessarily related to Swift.

**Comment by jTanG0506** [comment]
Had to resolve merge conflicts - @jtthrillson can you review again please? 🐯

**Pull request merged**